.github/workflows/draft_new_release.yml

```diff
@@ -0,0 +1,90 @@
+name: Draft new release
+
+on:
+  workflow_dispatch:
+    inputs:
+      module:
+        required: true
+
+jobs:
+  draft-new-release:
+    name: Draft a new release
+    runs-on: ubuntu-latest
+#    if: startsWith(github.ref, 'refs/heads/develop-v2') || startsWith(github.ref, 'refs/heads/hotfix/')
```

Comment:
no check for source branch?

Reply:
Sorry, this was to test with act. I will fix this.